### PR TITLE
Daniel code review

### PR DESCRIPTION
**Pull request by Dabesu**
After reviewing your code I found little to no error with it and just found things that could be changed but is all preference.
-Internal 
-Key Wording
-Code that did not play a role.
again, these are just small changes but if you have any questions let me know.

### CODE REVIEW

**Comment by Dabesu** [comment]
Here are some things I learned and changed accordingly because of it: Internal was added for it flexibility with this project, The unit test was removed because it didn't seem to have any use at the moment, and wording change was so it is easier to understand!

File: app/src/androidTest/java/com/example/foodjuju/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ import org.junit.Assert.*
 @RunWith(AndroidJUnit4::class)
 class ExampleInstrumentedTest {
     @Test
-    fun useAppContext() {
+    internal fun useAppContext() {
```
Comment (discospiff) [inline app/src/androidTest/java/com/example/foodjuju/ExampleInstrumentedTest.kt:19]: Good call... internal better than leaving it at the default (public)

File: app/src/main/java/com/example/foodjuju/ui/calendar/CalendarViewModel.kt [hunk 2]
```diff
@@ -7,7 +7,7 @@ import androidx.lifecycle.ViewModel
 class CalendarViewModel : ViewModel() {
 
     private val _text = MutableLiveData<String>().apply {
-        value = "This is dashboard Fragment"
+        value = "This is a dashboard Fragment"
```
Comment (discospiff) [inline app/src/main/java/com/example/foodjuju/ui/calendar/CalendarViewModel.kt:10]: That looks more readable.

File: app/src/test/java/com/example/foodjuju/ExampleUnitTest.kt [hunk 3]
```diff
@@ -10,8 +10,4 @@ import org.junit.Assert.*
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class ExampleUnitTest {
-    @Test
-    fun addition_isCorrect() {
```
Comment (discospiff) [inline app/src/test/java/com/example/foodjuju/ExampleUnitTest.kt:14]: Yeah... if it's not used, it's technical debt, which should be removed.

**Comment by discospiff** [comment]
Several good changes here that reduce technical debt.  I recommend merging these changes.

**Pull request merged**